DuckDuckGo/AppDelegate.swift

```diff
@@ -82,13 +82,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     private func startOnboardingFlowIfNotSeenBefore() {
+        
         var settings = TutorialSettings()
         if !settings.hasSeenOnboarding {
+            startMigration()
```

Comment:
I'm 80% sure that the onboaridng is getting deleted. From what I saw the migration was pretty fast so I'm ok with deferring any migration related ui (if you are too!) until we know we need it. Only caveat is that the migration needs to be non-destructive to any new bookmarks created while it is running.